lib/pubid/iso/transformer.rb

```diff
@@ -56,6 +56,8 @@ def self.convert_stage(code)
         "PRF"
       when "pD", "PD"
         "CD"
+      when "AWI"
+        "WD"
```

Comment:
@ronaldtse Is that ok? I know WD != AWI, but it has the same stage number "20.00"

Reply:
## AWI is stage 20.00

<img width="690" alt="Screenshot 2022-03-11 at 12 09 55 AM" src="https://user-images.githubusercontent.com/11865/157705354-020827ab-e023-46bb-a485-c9f55862bf3e.png">

## WD is stage 20.20 or 20.60 (i.e. 20.20+)

<img width="692" alt="Screenshot 2022-03-11 at 12 09 10 AM" src="https://user-images.githubusercontent.com/11865/157705099-dd8d0fc6-7475-4e54-9306-3fc6b505689e.png">

<img width="689" alt="Screenshot 2022-03-11 at 12 08 54 AM" src="https://user-images.githubusercontent.com/11865/157705020-6921aa50-5387-4a74-bd76-dc5c4c87740a.png">

## FDIS is 50.00+, but PRF can also be 50.00+

PRF and FDIS cannot really be distinguished.

PRFs being 50.00, 50.20:
<img width="706" alt="Screenshot 2022-03-11 at 12 16 49 AM" src="https://user-images.githubusercontent.com/11865/157706900-a54afdf5-2811-4344-bcff-d53ed3737b36.png">
<img width="679" alt="Screenshot 2022-03-11 at 12 17 31 AM" src="https://user-images.githubusercontent.com/11865/157707020-8b9e4986-44d8-435b-a850-612e0179df11.png">
<img width="677" alt="Screenshot 2022-03-11 at 12 18 13 AM" src="https://user-images.githubusercontent.com/11865/157707139-6cb044a7-5af6-41cc-81c6-fc25ccbf5c0e.png">

FDIS being 50.00, 50.20:
<img width="681" alt="Screenshot 2022-03-11 at 12 16 58 AM" src="https://user-images.githubusercontent.com/11865/157706932-0ae5307e-ddce-4a64-ab37-49357813dfc2.png">
<img width="684" alt="Screenshot 2022-03-11 at 12 18 58 AM" src="https://user-images.githubusercontent.com/11865/157707271-f59553fc-81aa-4983-abcb-81ddd57d2d61.png">